app/assets/javascripts/code/controllers/_mobile_sessions_map_ctrl.js

```diff
@@ -14,8 +14,7 @@ export const MobileSessionsMapCtrl = (
   functionBlocker,
```

Comment:
Moved all the logic to sessions_list_ctrl (it will make more sense later when fixed will have the crowd layer and crowd map will be removed)